.github/workflows/ci-build-test.yaml

```diff
@@ -0,0 +1,58 @@
+name: Build Master Release versions
+
+on:
+  push:
+    branches:
+      - main
+
+## Publish a docker image with the CLI? Future use as a runjob maybe?
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: aurae-runtime/ae
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    ## needed to publish eventually to GHCR
+    permissions:
+      contents: read
+      packages: write
+    defaults:
+      run:
+        shell: bash
+        working-directory: .
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Build
+        uses: docker/setup-buildx-action@v2
+
+      - name: Get version 
+        id: get_version
+        run:
+          echo "::set-output name=VERSION::$(make version)"
```

Comment:
This way of passing information between workflow steps is deprecated. Use the newer format instead. See https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/